build.js

```diff
@@ -3,16 +3,19 @@ console.log('Before running, make sure versions are updated in both package.json
 
 var NwBuilder = require('node-webkit-builder');
 var appPkg = require('./desktop-app/package.json');
-var appName = 'Tinder⁺⁺';
+var appName = 'Tinder Desktop';
```

Comment:
`var appName = 'Tinder Desktop';` is probably internal too so can be tinder-desktop, but then the paths in installer-script.iss containing 'Tinder Desktop' need to be renamed accordingly.